### PR TITLE
Fix setting accuracy

### PR DESCRIPTION
**Pull request by Michal-Szczepaniak**
So, for some reason, after `m_currentLocation = location;` accuracy becones NaN. I tried doing `m_currentLocation.setAccuracy(location.accuracy())` but the accuracy was still NaN. I tried `m_currentLocation.accuracy().setHorizontal(location.accuracy().horizontal());` but that was NaN as well. This solution thankfully works. If you want to debug it further and find real reason, feel free, I would just like to have working accuracy asap

### CODE REVIEW

File: hybrisprovider.cpp [hunk 1]
```diff
@@ -520,6 +520,10 @@ void HybrisProvider::setLocation(const Location &location)
     }
 
     m_currentLocation = location;
+    Accuracy acc;
+    acc.setHorizontal(location.accuracy().horizontal());
+    acc.setVertical(location.accuracy().vertical());
+    m_currentLocation.setAccuracy(acc);
```
Comment (rainemak) [inline hybrisprovider.cpp:526]: This looks to be an issue within the copy constructor of the LocationData (locationtypes.h). 
Adding following to the 58 of the locationtypes fixes copy constructor.

```
direction(other.direction), climb(other.climb), accuracy(other.accuracy)
```
So, accuracy after climb.

Comment (Michal-Szczepaniak) [reply hybrisprovider.cpp:526]: I will test that for you if you need me to

Comment (rainemak) [reply hybrisprovider.cpp]: Awesome, thank you

Comment (Michal-Szczepaniak) [reply hybrisprovider.cpp]: @rainemak It seems that's it's working! To remove my previous commit you can squash changes while merging :)

Comment (rainemak) [reply hybrisprovider.cpp]: Thank you @Michal-Szczepaniak !

**Review by rainemak — COMMENTED** [review]
Approved

**Pull request merged**